microscopium/preprocess.py

```diff
@@ -808,6 +808,50 @@ def montage(ims, order=None):
     return montaged
 
 
+def find_missing_fields(fns, order=None,
+                        re_string=".*_[A-P]\d{2}f(\d{2})d0",
```

Comment:
Should this be a Python raw string, `r'mystring\with\special\chars'`? Prevents involuntary escapes.